webapp/src/components/home/Home.vue

```diff
@@ -9,17 +9,17 @@
         href="https://framaforms.org/disco-feedback-form-1718716636"
         >
         <div class="flex flex-row flex-wrap shrink-0 items-center gap-x-2 justify-end">
-            <p class="text-disco-blue font-bold text-xs hover:underline hover:text-disco-cyan text-end">Give us some feedback</p>
+            <p class="text-disco-blue font-bold text-xs hover:underline hover:text-disco-cyan text-end dark:text-disco-dark-cyan">Give us some feedback</p>
           <FeedbackIcon custom-class="min-w-8 min-h-8 w-8 h-8"/>
         </div>
       </a>
   </div>
     <!-- Disco logo -->
     <div class="flex flex-col justify-center items-center mb-8 space-y-4">
       <div class="md:max-w-md lg:max-w-lg">
-        <DiscoGIF/>
+        <DiscoGIF class="bg-slate-100 rounded-lg pb-16 px-4"/>
```

Comment:
```suggestion
        <DiscoGIF class="bg-slate-100 rounded-lg pb-4 px-4"/>
```
I think it looks a bit weird in the light theme to have such a big space (it was my idea I know). We could change the padding with the dark theme but it doesn't look very nice either so maybe just decrease it?

Reply:
I changed it to 8, so that the logo could respire.